.travis.yml

```diff
@@ -3,6 +3,7 @@ cache:
   packages: true
   directories:
     - $HOME/.TinyTeX
+warnings_are_errors: true
```

Comment:
Please refrain from making irrelevant changes or implementing several new features in one PR (we could discuss Appveyor in another PR): https://yihui.name/en/2018/02/bite-sized-pull-requests/

This option doesn't really need to be set since `true` is the default: https://docs.travis-ci.com/user/languages/r/#package-check-options

Reply:
Ah, didn't realize, reverted.  Its been hard to find the time to refactor this into smaller ones.

Reply:
Thanks for the comments.  Think this is resolved.